ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -119,6 +119,66 @@ function suite() {
 echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_TEST_START"
 trap 'echo "$(date +%s)" > "${SHARED_DIR}/TEST_TIME_TEST_END"' EXIT
 
+oc -n openshift-config patch cm admin-acks --patch '{"data":{"ack-4.8-kube-1.22-api-removals-in-4.9":"true"}}' --type=merge || echo 'failed to ack the 4.9 Kube v1beta1 removals; possibly API-server issue, or a pre-4.8 release image'
+
+# wait for ClusterVersion to level, until https://bugzilla.redhat.com/show_bug.cgi?id=2009845 makes it back to all 4.9 releases being installed in CI
+oc wait --for=condition=Progressing=False --timeout=2m clusterversion/version
+
+# wait for all clusteroperators to reach progressing=false to ensure that we achieved the configuration specified at installation
+# time before we run our e2e tests.
+echo "$(date) - waiting for clusteroperators to finish progressing..."
+oc wait clusteroperators --all --for=condition=Progressing=false --timeout=10m
```

Comment:
It wouldn't be any harm to add this command twice, I've seen 'co' go from Progressing=true too Progressing=false 
and back again, doing two passes would have a higher chance of catching this, if its not needed the second pass will only take a few seconds.

Reply:
Without waiting for some time between two checks? Otherwise we'll grab only those co that immediately change state after the first check succeeded

Reply:
They don't have to immediately change after the first check succeeded, any CO may have changed long before the command returned, as the command seems to check each one sequentially. i.e. if the first one on the list wasn't Progressing and the command is waiting on the 10th on the list to finish Progressing. It doesn't go back a recheck the first one when the 10th on is finished.

Reply:
Should we check for Progressing=False and Available=True ?

Reply:
> Should we check for Progressing=False and Available=True ?

 This would be ideal but doesn't look like it works, I tried 
`oc wait clusteroperators --all --for=condition=Progressing=false --timeout=10m --for=condition=Available=True`

but it passed straight through even with kube-apiserver with Progressing=True
looks like its not supported https://github.com/kubernetes/kubernetes/issues/95759